website/server/models/subscriptionPlan.js

```diff
@@ -16,6 +16,9 @@ export let schema = new mongoose.Schema({
   gemsBought: {type: Number, default: 0},
   mysteryItems: {type: Array, default: () => []},
   lastBillingDate: Date, // Used only for Amazon Payments to keep track of billing date
+  additionalData: mongoose.Schema.Types.Mixed,
```

Comment:
This is supposed to have whatever structure/schema the payment method needs. I could document the structure for the google subscriptions, but I'm not sure if that makes sense, if it's going to be different for each payment method.